aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -286,8 +294,12 @@ spec aptos_framework::account {
         aborts_if account_scheme != ED25519_SCHEME && account_scheme != MULTI_ED25519_SCHEME;
 
         modifies global<Account>(source_address);
+        let post offer_for = global<Account>(source_address).rotation_capability_offer.for;
+        ensures option::spec_borrow(offer_for) == recipient_address;
     }
 
+
+
```

Comment:
nit: remove one of the new lines and leave just one?

Reply:
fixed